lib/workers/repository/process/lookup/rollback.ts

```diff
@@ -38,7 +40,16 @@ export function getRollbackUpdate(
     'Versions found before rolling back'
   );
   lessThanVersions.sort((a, b) => version.sortVersions(a.version, b.version));
-  const newVersion = lessThanVersions.pop()?.version;
+  let stableVersions;
+  let newVersion;
+  if (ignoreUnstable) {
+    stableVersions = lessThanVersions.filter((v) =>
+      isVersionStable(v.version, lessThanVersions, version)
```

Comment:
```suggestion
    stableVersions = lessThanVersions.filter(v =>
     version.isStable(v.version));
```
Could `isStable` be enough here?